pipeline/03-evaluate.R

```diff
@@ -316,70 +305,72 @@ gen_agg_stats_quantile <- function(data, truth, estimate,
 
 # Use fancy tidyeval to create a list of all the geography levels with a
 # class or no class option for each level
-geographies_quosures <- rlang::quos(
-  meta_township_code,
-  meta_nbhd_code, loc_cook_municipality_name,
-  loc_ward_num, loc_census_puma_geoid, loc_census_tract_geoid,
-  loc_school_elementary_district_geoid, loc_school_secondary_district_geoid,
-  loc_school_unified_district_geoid,
-  NULL
+geographies_quosures <- c(
+  rlang::parse_quos(params$ratio_study$geographies, env = .GlobalEnv),
+  rlang::quo(NULL)
 )
-geographies_list <- purrr::cross2(
+geographies_list <- unname(as.list(tidyr::expand_grid(
   geographies_quosures,
   rlang::quos(meta_class, NULL)
-)
+)))
```

Comment:
**[Question, non-blocking]** As someone who's still pretty new to tidyverse syntax, I'm curious if there's an advantage to writing nested calls this way instead of by piping the output?

```R
geographies_list <- geographies_quosures %>%
  tidyr::expand_grid(rlang::quos(meta_class, NULL)) %>%
  as.list() %>%
  unnest()
```

Reply:
There is not! Fixed in 56babdf.